<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.2.313">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Raquel Francisco and Seth Latter">
<meta name="dcterms.date" content="2023-04-07">

<title>A Match Made in Landfills?</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1.6em;
  vertical-align: middle;
}
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>


<script src="Manuscript_files/libs/clipboard/clipboard.min.js"></script>
<script src="Manuscript_files/libs/quarto-html/quarto.js"></script>
<script src="Manuscript_files/libs/quarto-html/popper.min.js"></script>
<script src="Manuscript_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="Manuscript_files/libs/quarto-html/anchor.min.js"></script>
<link href="Manuscript_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="Manuscript_files/libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="Manuscript_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="Manuscript_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="Manuscript_files/libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">


</head>

<body class="fullcontent">

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article">

<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">A Match Made in Landfills?</h1>
<p class="subtitle lead">Exploring the diversity and burden of antimicrobial resistance genes carried by white stork (Ciconia ciconia) throughout the breeding season in Madrid, Spain</p>
</div>



<div class="quarto-title-meta">

    <div>
    <div class="quarto-title-meta-heading">Author</div>
    <div class="quarto-title-meta-contents">
             <p>Raquel Francisco and Seth Latter </p>
          </div>
  </div>
    
    <div>
    <div class="quarto-title-meta-heading">Published</div>
    <div class="quarto-title-meta-contents">
      <p class="date">April 7, 2023</p>
    </div>
  </div>
  
    
  </div>
  

</header>

<div class="cell">

</div>
<section id="highlights" class="level1">
<h1>Highlights</h1>
<ul>
<li><p>Our results suggest that landfills may not be as impactful as previously believed to the emergence and maintenance of AMR in this system.</p></li>
<li><p>Storks are most impacted by multi-drug resistance and ARG burden when anthropogenic waste is most heavily consumed later in the breeding season.</p></li>
<li><p>This species appears to be a good sentinel for anthropogenic impact on environment.</p></li>
<li><p>Future efforts in stork AMR research should focus on exploring the relationship between other anthropogenic environments (e.g., agricultural pastures) and health.</p></li>
</ul>
</section>
<section id="abstract" class="level1">
<h1>Abstract</h1>
<p>Anthropogenic environments are hotspots for the emergence and maintenance of antimicrobial resistance. Agricultural pastures and landfills are of particular interest due to their complex microbial communities and abundant wildlife visitation, which could facilitate the exchange of antimicrobial resistance genes (ARGs) via horizontal transfer. Wild birds that occupy these environments may become both reservoirs and transporters of ARGs. The White Stork is a highly urbanized wading bird that has significantly changed its ecology due to shifts in Spanish waste management toward open-air landfills. This species now heavily forages at landfills, which provide abundant food and allow for improved reproductive success. Birds that are dependent on anthropogenic resources, such as storks, provide the ideal opportunity to understand the emergence and spread of AMR. We evaluated the diversity and quantity of ARGs in storks during three periods of the breeding season (defined by distinct foraging strategies). A total of 31 nests at Prado Herrero were sampled between March-July (2020-2021). Fresh feces were collected from 31 nests to evaluate the presence of 23 important ARGs affecting eight antibiotic classes via quantitative PCR. All nests carried multiple ARGs. Over 70% of nests had multi-drug resistance to at least 3 antibiotic classes during at least one time-period. Generalized Linear Mixed Models revealed that increased diversity in antibiotic class resistance, amount of ARGs present in a sample, and multi-drug resistance were associated with increased adult age and decreased landfill use. Our results suggest that landfills may not be contributing significantly to the emergence and maintenance of AMR in this system. Little literature exists on the relationship between stork habitat selection and health outside of landfill use in Spain. Future efforts in stork AMR research should focus on exploring the relationship between agricultural land use and health.</p>
<div style="page-break-after: always;"></div>
</section>
<section id="introduction" class="level1">
<h1>Introduction</h1>
<section id="general-background-information" class="level2">
<h2 class="anchored" data-anchor-id="general-background-information">General Background Information</h2>
<p><em>Provide enough background on your topic that others can understand the why and how of your analysis</em></p>
</section>
<section id="description-of-data-and-data-source" class="level2">
<h2 class="anchored" data-anchor-id="description-of-data-and-data-source">Description of data and data source</h2>
<p><em>We have 126 observations taken over a period of 2 years. We are evaluating for the presence and burden of antimicrobial resistance genes in white stork feces. Codebook is a WIP located in raw data folder.</em></p>
</section>
<section id="questionshypotheses-to-be-addressed" class="level2">
<h2 class="anchored" data-anchor-id="questionshypotheses-to-be-addressed">Questions/Hypotheses to be addressed</h2>
<p>Does landfill use by storks increase the likelihood of carrying Antimicrobial resistance (AMR) genes and resistance gene burden?.</p>
<p>Stork that visit the landfill more often will have a higher likelihood of carrying AMR genes and have a higher burden of AMR genes.</p>
<p>Is nest success related to AMR gene burden and multi-drug resistance (i.e., resistance to 3 or more drug classes)?</p>
<p>Stork with more AMR gene burdens and multi-drug resistance will have lower nest success.</p>
<p><em>To cite other work (important everywhere, but likely happens first in introduction), make sure your references are in the bibtex file specified in the YAML header above (here <code>dataanalysis_template_references.bib</code>) and have the right bibtex key. Then you can include like this:</em></p>
<p>Examples of reproducible research projects can for instance be found in <span class="citation" data-cites="mckay2020 mckay2020a">(<a href="#ref-mckay2020" role="doc-biblioref">McKay, Ebell, Billings, et al., 2020</a>; <a href="#ref-mckay2020a" role="doc-biblioref">McKay, Ebell, Dale, Shen, &amp; Handel, 2020</a>)</span></p>
<div style="page-break-after: always;"></div>
</section>
</section>
<section id="methods" class="level1">
<h1>Methods</h1>
<section id="data-aquisition" class="level2">
<h2 class="anchored" data-anchor-id="data-aquisition">Data aquisition</h2>
<section id="experimental-design" class="level3">
<h3 class="anchored" data-anchor-id="experimental-design">Experimental Design</h3>
<p><em>Study Area</em> — This study took place in Prado Herrero, a private cattle ranch located northwest of metropolitan Madrid and is surrounded by agriculture (e.g.&nbsp;beef cattle, cereal grains, legumes, and forage plants (“Renta Agraria,” 2019)). Prado Herrero is located within a nationally protected area (Cuenca Alta del Manzanares Regional Park) and is just north of Santillana Reservoir. This reservoir that was declared an Important Bird Area by Regional Catalogue of Reservoirs and Wetlands of the Community of Madrid due to the numerous resident and migratory species that utilize this water source <cite>. This cattle ranch has supported a productive white stork rookery where storks have been banded and monitored by biologists at UCM for over 20 years &lt;CITE: Aguirre and Atienza&gt;. During the 2020 to 2021 breeding season, between the months of March to June, stork nests were identified, marked, and monitored for productivity. Of marked nests between the 2020 and 2021 breeding seasons, 31 with banded adults were used both years to lay eggs successfully. All 31 nests were located in ash trees (Fraxinus angustifolius ) found within the cattle pasture. Storks that breed within the Prado Herrero rookery are known to utilize Colmenar Viejo Landfill which is located approximately 12km southeast. Colmenar Viejo is an open-air landfill and it is second largest of it’s kind in the Madrid region &lt;CITE: Alejandro 2022&gt;.</cite></p>
<p><em>Sample Collection</em> — Between March to May 2020 and 2021, we collected feces from marked nests with banded adults in known breeding pairs at 3 points of the breeding season; (1) an adult sample during incubation, (2) an early juvenile sample during the first two weeks of the chicks life when adults are believed to forage on natural sources &lt;CITE: &gt;, and (3) a late juvenile sample after chicks were past two weeks of age when adults forage on anthropogenic resources. Nests were visited in the late mornings and approximately one gram of fresh feces was collected from the perimeter of the nest structure into a sterile Eppendorf tube (). Samples were maintained cold in a portable cooler with frozen gel packs and frozen in a −20°C freezer within 4 hours of collection and processed at a later date.</p>
<p><em>Ethics statement</em>: All animal handling was authorized by Cumunidad de Madrid: Consejeria de Medio Ambiente, Administracion Local y Ordenacion de Territorio. The permit number is D.N.I. nº 07.239.972-D.</p>
</section>
<section id="molecular-analysis-of-args" class="level3">
<h3 class="anchored" data-anchor-id="molecular-analysis-of-args">Molecular analysis of ARGs</h3>
<p>We performed total DNA extraction directly from fecal samples, by using a pressure filtration technique (QuickGene DNA Tissue Kit S, Fujifilm, Japan) following the manufacturer’s instructions. The 16S rRNA gene was amplified in each DNA sample by real time PCR (rtPCR) in 10-fold dilutions of extracted samples, according to Jiang et al.&nbsp;(2013)<cite:fix>. A DNA sample was considered validated when a ten-fold dilution showed a cycle threshold (Ct) less than 25 (Esperón et al., 2018). Once validated, we analyzed DNA samples by with a panel of 21 different ARGs encoding resistance to eight different antimicrobial classes: tetracyclines (tet(A), tet(B), tet(Y), tet(K), tet(M), tet(Q), tet(S), and tet(W)), sulfonamides (sulI and sulII), aminoglycosides (str and aadA), phenicols (catI and catII), macrolides (ermB and ermF), quinolones (qnrS and qnrB), betalactams (blaTEM and mecA), and polymyxins (mcr-1). All rtPCR reactions utilized premade gelled format 96-well plates (Biotools, B &amp;M Labs, S.A., Madrid, Spain), with the exception of blaTEM and mecA genes which used the Sybr GreenTM and TaqManTM probe, respectively. Our thermal cycle was the same for all the rtPCR reactions [6′ 95 °C, 40× (10″ 95 °C, 30″ 60 °C)], with alignment and extension in the same step, at constant temperature of 60 °C. A melting curve step was performed at the end of the qPCR reaction to validate the authenticity of the positive (Nieto-Claudin et al., 2019). We quantified the relative burden of each gene for each sample via the cycle threshold (Ct) for the 16S rRNA and the Ct value of the individual ARG using a previously published formula (Esperón et al., 2020).</cite:fix></p>
</section>
</section>
<section id="data-import-and-cleaning-reference-file-and-give-details-here.-reference-a-readmedoc-here" class="level2">
<h2 class="anchored" data-anchor-id="data-import-and-cleaning-reference-file-and-give-details-here.-reference-a-readmedoc-here">Data import and cleaning (Reference file and give details HERE. Reference a ReadME/doc here)</h2>
<p><em>Write code that reads in the file and cleans it so it’s ready for analysis. Since this will be fairly long code for most datasets, it might be a good idea to have it in one or several R scripts. If that is the case, explain here briefly what kind of cleaning/processing you do, and provide more details and well documented code somewhere (e.g.&nbsp;as supplement in a paper). All materials, including files that contain code, should be commented well so everyone can follow along.</em></p>
<div class="cell">
<div class="sourceCode cell-code" id="cb1"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(tidyverse)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>── Attaching core tidyverse packages ──────────────────────── tidyverse 2.0.0 ──
✔ dplyr     1.1.0     ✔ readr     2.1.4
✔ forcats   1.0.0     ✔ stringr   1.5.0
✔ ggplot2   3.4.1     ✔ tibble    3.1.8
✔ lubridate 1.9.2     ✔ tidyr     1.3.0
✔ purrr     1.0.1     
── Conflicts ────────────────────────────────────────── tidyverse_conflicts() ──
✖ dplyr::filter() masks stats::filter()
✖ dplyr::lag()    masks stats::lag()
ℹ Use the conflicted package (&lt;http://conflicted.r-lib.org/&gt;) to force all conflicts to become errors</code></pre>
</div>
<div class="sourceCode cell-code" id="cb3"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb3-1"><a href="#cb3-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(here)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<section id="statistical-analysis" class="level3">
<h3 class="anchored" data-anchor-id="statistical-analysis">Statistical analysis</h3>
<p>Presence absence ARG results obtained from the fecal samples between 2020 to 2021 were used for simple summary statistics. Samples were classified as “multiresistant” if they were resistant to three or more of the 8 antibiotic classes that we evaluated for in this study (Blanco-Peña et al., 2017). In addition, we applied the following formula to estimate the percentage of bacteria harboring ARGs: x = 10[2+0.33(ct16S-ctARG)], where x individual percent gene burden in the sample (i.e., the estimated number of copies of the gene present per reaction). Results were expressed in log10, ranging from −8 ( zero to a negligible amount of the bacteria in the sample carried an ARG) to 2 (all or 100% of the bacteria in the sample carried an ARG). The inverse Log10 was then applied to results so they could be totalled and used to evalute total gene burdens accross sampling periods.</p>
<p>Several linear mixed models (LMM) were constructed to evaluate multi-resistance and ARG burden as response variables with nest as a random factor. Covariates considered with each response variable included adult age, adult mean land fill use index (LUI), sample period (as described above), and nest success. Landfill use was quantified by physically observing a banded stork at Colmenar Viejo during weekly visits from March to June in 2021. The LUI was calculated as the number of observations of one particular bird within the total number of visits to landfill per year &lt;CITE Alejandro 2022&gt;. If a banded adult was not seen at the landfill during the breeding season, they were assigned a LUI of 0, suggestive of no resource provisioning at the landfill. All covariates were evaluated for correlation, no covariates were correlated with all the Spearman’s correlation coefficients (rho) &lt; 0.5 and the p &gt; 0.05. All continuous variables (LUI, age, and nest success) were then standardized prior to analysis.</p>
<p>All models were constructed with only 2021 data, as the COVID-19 pandemic prohibited the collection of LUI data in 2020. Models were built and fitted to data using the statistical package tidymodels in Program R (R version 4.2.1, www.r-project.org).</p>
<section id="note-still-trying-to-figure-out-how-to-do-this-with-the-performance-package-we-evaluated-model-fit-for-all-models-using-akaike-information-criterion-scores-adjusted-for-small-sample-sizes-aicc-burnham-and-anderson-2002.-the-best-fit-model-was-selected-by-having-a-δaicc-2._" class="level4">
<h4 class="anchored" data-anchor-id="note-still-trying-to-figure-out-how-to-do-this-with-the-performance-package-we-evaluated-model-fit-for-all-models-using-akaike-information-criterion-scores-adjusted-for-small-sample-sizes-aicc-burnham-and-anderson-2002.-the-best-fit-model-was-selected-by-having-a-δaicc-2._">(Note: still trying to figure out how to do this with the performance package) We evaluated model fit for all models using Akaike Information Criterion scores adjusted for small sample sizes (AICc: Burnham and Anderson, 2002). The best fit model was selected by having a ΔAICc &lt; 2._</h4>
<div style="page-break-after: always;"></div>
</section>
</section>
</section>
</section>
<section id="results" class="level1">
<h1>Results</h1>
<section id="exploratorydescriptive-analysis-summary-stats-in-exploration-file-tbd---burdenmdr-associated-with-each-class.-summary-table-with-actual-summary-stats" class="level2">
<h2 class="anchored" data-anchor-id="exploratorydescriptive-analysis-summary-stats-in-exploration-file-tbd---burdenmdr-associated-with-each-class.-summary-table-with-actual-summary-stats">Exploratory/Descriptive analysis (SUMMARY STATS IN EXPLORATION FILE TBD -&gt; Burden/MDR associated with each class. summary table? with actual summary stats)</h2>
<p><em>Use a combination of text/tables/figures to explore and describe your data. Show the most important descriptive results here. Additional ones should go in the supplement. Even more can be in the R and Quarto files that are part of your project.</em></p>
<p><strong>?@tbl-summarytable</strong> shows a summary of the data.</p>
<p>Note the loading of the data providing a <strong>relative</strong> path using the <code>../../</code> notation. (Two dots means a folder up). You never want to specify an <strong>absolute</strong> path like <code>C:\ahandel\myproject\results\</code> because if you share this with someone, it won’t work for them since they don’t have that path. You can also use the <code>here</code> R package to create paths. See examples of that below.</p>
<div id="tbl-summarytable" class="cell tbl-parent quarto-layout-panel anchored" data-tbl-cap="Data summary table.">

</div>
</section>
<section id="basic-statistical-analysis" class="level2">
<h2 class="anchored" data-anchor-id="basic-statistical-analysis">Basic statistical analysis</h2>
<p><em>To get some further insight into your data, if reasonable you could compute simple statistics (e.g.&nbsp;simple models with 1 predictor) to look for associations between your outcome(s) and each individual predictor variable. Though note that unless you pre-specified the outcome and main exposure, any “p&lt;0.05 means statistical significance” interpretation is not valid.</em></p>
<p><strong>?@fig-result</strong> shows a scatterplot figure produced by one of the R scripts.</p>
<div class="cell">
<div class="cell-output-display">
<div id="fig-result-1" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="../../3%20Analysis%20Scripts/LUIvsBurden_LME.tiff" class="img-fluid figure-img"></p>
<p></p><figcaption class="figure-caption">Figure&nbsp;1: Height and weight stratified by gender.</figcaption><p></p>
</figure>
</div>
</div>
<div class="cell-output-display">
<div id="fig-result-2" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="../../3%20Analysis%20Scripts/LUIvsMDR_LME.tiff" class="img-fluid figure-img"></p>
<p></p><figcaption class="figure-caption">Figure&nbsp;2: Height and weight stratified by gender.</figcaption><p></p>
</figure>
</div>
</div>
</div>
</section>
<section id="full-analysis" class="level2">
<h2 class="anchored" data-anchor-id="full-analysis">Full analysis</h2>
<p><em>Use one or several suitable statistical/machine learning methods to analyze your data and to produce meaningful figures, tables, etc. This might again be code that is best placed in one or several separate R scripts that need to be well documented. You want the code to produce figures and data ready for display as tables, and save those. Then you load them here.</em></p>
<p>Example <strong>?@tbl-resulttable2</strong> shows a summary of a linear model fit.</p>
<div id="tbl-resulttable2" class="cell tbl-parent quarto-layout-panel anchored">
<div class="panel-caption table-caption">
<p>Linear model fit table.</p>
</div>
<div class="quarto-layout-row quarto-layout-valign-top">
<div class="cell-output cell-output-stdout quarto-layout-cell" style="flex-basis: 100.0%;justify-content: center;">
<pre><code>               Name     Model      AICc_wt Performance_Score
1      glmer_fit_11 _glmerMod 3.254465e-01      1.000000e+00
2       glmer_fit_2 _glmerMod 3.163397e-01      9.720177e-01
3      glmer_fit_12 _glmerMod 1.173421e-01      3.605572e-01
4  glmer_fit_global _glmerMod 9.626812e-02      2.958031e-01
5      glmer_fit_10 _glmerMod 5.617098e-02      1.725966e-01
6       glmer_fit_9 _glmerMod 2.565916e-02      7.884285e-02
7       glmer_fit_8 _glmerMod 2.515228e-02      7.728536e-02
8       glmer_fit_1 _glmerMod 1.927419e-02      5.922374e-02
9       glmer_fit_5 _glmerMod 1.125752e-02      3.459091e-02
10      glmer_fit_6 _glmerMod 7.089238e-03      2.178302e-02
11   glmer_fit_null _glmerMod 9.237380e-08      1.879885e-07
12      glmer_fit_3 _glmerMod 7.243553e-08      1.267242e-07
13      glmer_fit_4 _glmerMod 3.605959e-08      1.495174e-08
14      glmer_fit_7 _glmerMod 3.119360e-08      0.000000e+00</code></pre>
</div>
</div>
<div class="quarto-layout-row quarto-layout-valign-top">
<div class="cell-output-display quarto-layout-cell" style="flex-basis: 100.0%;justify-content: center;">
<table class="table table-sm table-striped">
<thead>
<tr class="header">
<th style="text-align: left;">Name</th>
<th style="text-align: left;">Model</th>
<th style="text-align: right;">AICc_wt</th>
<th style="text-align: right;">Performance_Score</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">glmer_fit_11</td>
<td style="text-align: left;">_glmerMod</td>
<td style="text-align: right;">0.3254465</td>
<td style="text-align: right;">1.0000000</td>
</tr>
<tr class="even">
<td style="text-align: left;">glmer_fit_2</td>
<td style="text-align: left;">_glmerMod</td>
<td style="text-align: right;">0.3163397</td>
<td style="text-align: right;">0.9720177</td>
</tr>
<tr class="odd">
<td style="text-align: left;">glmer_fit_12</td>
<td style="text-align: left;">_glmerMod</td>
<td style="text-align: right;">0.1173421</td>
<td style="text-align: right;">0.3605572</td>
</tr>
<tr class="even">
<td style="text-align: left;">glmer_fit_global</td>
<td style="text-align: left;">_glmerMod</td>
<td style="text-align: right;">0.0962681</td>
<td style="text-align: right;">0.2958031</td>
</tr>
<tr class="odd">
<td style="text-align: left;">glmer_fit_10</td>
<td style="text-align: left;">_glmerMod</td>
<td style="text-align: right;">0.0561710</td>
<td style="text-align: right;">0.1725966</td>
</tr>
<tr class="even">
<td style="text-align: left;">glmer_fit_9</td>
<td style="text-align: left;">_glmerMod</td>
<td style="text-align: right;">0.0256592</td>
<td style="text-align: right;">0.0788428</td>
</tr>
<tr class="odd">
<td style="text-align: left;">glmer_fit_8</td>
<td style="text-align: left;">_glmerMod</td>
<td style="text-align: right;">0.0251523</td>
<td style="text-align: right;">0.0772854</td>
</tr>
<tr class="even">
<td style="text-align: left;">glmer_fit_1</td>
<td style="text-align: left;">_glmerMod</td>
<td style="text-align: right;">0.0192742</td>
<td style="text-align: right;">0.0592237</td>
</tr>
<tr class="odd">
<td style="text-align: left;">glmer_fit_5</td>
<td style="text-align: left;">_glmerMod</td>
<td style="text-align: right;">0.0112575</td>
<td style="text-align: right;">0.0345909</td>
</tr>
<tr class="even">
<td style="text-align: left;">glmer_fit_6</td>
<td style="text-align: left;">_glmerMod</td>
<td style="text-align: right;">0.0070892</td>
<td style="text-align: right;">0.0217830</td>
</tr>
<tr class="odd">
<td style="text-align: left;">glmer_fit_null</td>
<td style="text-align: left;">_glmerMod</td>
<td style="text-align: right;">0.0000001</td>
<td style="text-align: right;">0.0000002</td>
</tr>
<tr class="even">
<td style="text-align: left;">glmer_fit_3</td>
<td style="text-align: left;">_glmerMod</td>
<td style="text-align: right;">0.0000001</td>
<td style="text-align: right;">0.0000001</td>
</tr>
<tr class="odd">
<td style="text-align: left;">glmer_fit_4</td>
<td style="text-align: left;">_glmerMod</td>
<td style="text-align: right;">0.0000000</td>
<td style="text-align: right;">0.0000000</td>
</tr>
<tr class="even">
<td style="text-align: left;">glmer_fit_7</td>
<td style="text-align: left;">_glmerMod</td>
<td style="text-align: right;">0.0000000</td>
<td style="text-align: right;">0.0000000</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="quarto-layout-row quarto-layout-valign-top">
<div class="cell-output-display quarto-layout-cell" style="flex-basis: 100.0%;justify-content: center;">
<table class="table table-sm table-striped">
<thead>
<tr class="header">
<th style="text-align: left;">Name</th>
<th style="text-align: left;">Model</th>
<th style="text-align: right;">AICc_wt</th>
<th style="text-align: right;">Performance_Score</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">lmer_fit2_global</td>
<td style="text-align: left;">_lmerMod</td>
<td style="text-align: right;">0.2643531</td>
<td style="text-align: right;">1.0000000</td>
</tr>
<tr class="even">
<td style="text-align: left;">lmer_fit2_2</td>
<td style="text-align: left;">_lmerMod</td>
<td style="text-align: right;">0.2283685</td>
<td style="text-align: right;">0.8638768</td>
</tr>
<tr class="odd">
<td style="text-align: left;">lmer_fit2_11</td>
<td style="text-align: left;">_lmerMod</td>
<td style="text-align: right;">0.1340166</td>
<td style="text-align: right;">0.5069607</td>
</tr>
<tr class="even">
<td style="text-align: left;">lmer_fit2_12</td>
<td style="text-align: left;">_lmerMod</td>
<td style="text-align: right;">0.1128365</td>
<td style="text-align: right;">0.4268403</td>
</tr>
<tr class="odd">
<td style="text-align: left;">lmer_fit2_9</td>
<td style="text-align: left;">_lmerMod</td>
<td style="text-align: right;">0.1112876</td>
<td style="text-align: right;">0.4209809</td>
</tr>
<tr class="even">
<td style="text-align: left;">lmer_fit2_10</td>
<td style="text-align: left;">_lmerMod</td>
<td style="text-align: right;">0.0818244</td>
<td style="text-align: right;">0.3095268</td>
</tr>
<tr class="odd">
<td style="text-align: left;">lmer_fit2_8</td>
<td style="text-align: left;">_lmerMod</td>
<td style="text-align: right;">0.0673134</td>
<td style="text-align: right;">0.2546344</td>
</tr>
<tr class="even">
<td style="text-align: left;">lmer_fit2_5</td>
<td style="text-align: left;">_lmerMod</td>
<td style="text-align: right;">0.0000000</td>
<td style="text-align: right;">0.0000000</td>
</tr>
<tr class="odd">
<td style="text-align: left;">lmer_fit2_1</td>
<td style="text-align: left;">_lmerMod</td>
<td style="text-align: right;">0.0000000</td>
<td style="text-align: right;">0.0000000</td>
</tr>
<tr class="even">
<td style="text-align: left;">lmer_fit2_6</td>
<td style="text-align: left;">_lmerMod</td>
<td style="text-align: right;">0.0000000</td>
<td style="text-align: right;">0.0000000</td>
</tr>
<tr class="odd">
<td style="text-align: left;">lmer_fit2_null</td>
<td style="text-align: left;">_lmerMod</td>
<td style="text-align: right;">0.0000000</td>
<td style="text-align: right;">0.0000000</td>
</tr>
<tr class="even">
<td style="text-align: left;">lmer_fit2_3</td>
<td style="text-align: left;">_lmerMod</td>
<td style="text-align: right;">0.0000000</td>
<td style="text-align: right;">0.0000000</td>
</tr>
<tr class="odd">
<td style="text-align: left;">lmer_fit2_4</td>
<td style="text-align: left;">_lmerMod</td>
<td style="text-align: right;">0.0000000</td>
<td style="text-align: right;">0.0000000</td>
</tr>
<tr class="even">
<td style="text-align: left;">lmer_fit2_7</td>
<td style="text-align: left;">_lmerMod</td>
<td style="text-align: right;">0.0000000</td>
<td style="text-align: right;">0.0000000</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div style="page-break-after: always;"></div>
</section>
</section>
<section id="discussion" class="level1">
<h1>Discussion</h1>
<section id="summary-and-interpretation" class="level2">
<h2 class="anchored" data-anchor-id="summary-and-interpretation">Summary and Interpretation</h2>
<ul>
<li><p>As multi-drug resistance and class diversity increase (similar things I know) throughout the breeding season (compounding effect likely), mean LUI decreases and nest success decreases (makes sense because in your prior papers you have found increase nest success with increased LUI).</p></li>
<li><p>Resistance gene burden appears to increase as mean LUI use and age increase. o Most of the burden is due to blaTEM, a common resistance gene associated with anthropogenic impact. Sampling period does not appear to explain burden, but the top blaTEM model did show a trend in burden increasing from the 2nd sampling period to the 3rd sampling period and age.</p></li>
</ul>
</section>
<section id="strengths-and-limitations" class="level2">
<h2 class="anchored" data-anchor-id="strengths-and-limitations">Strengths and Limitations</h2>
<ul>
<li>Not much statistically significant data, thus we may have to argue biological significance.</li>
</ul>
</section>
<section id="conclusions" class="level2">
<h2 class="anchored" data-anchor-id="conclusions">Conclusions</h2>
<p>LUI appears to be correlated with higher levels of AMR gene burden in storks. As LUI increases thorough the breeding season (Bialas et al 2021) resistance gene burden also increases with beta lactam resistance contributing to the majority of the burden. However, multidrug resistance appears to decrease as LUI increases, thus it is likely that storks are being exposed to antimicrobial resistance genes at other foraging areas (urban centers, agricultural pastures, etc.). Our results suggest that landfills may not be contributing significantly to the emergence and maintenance of AMR in this system. Little literature exists on the relationship between stork habitat selection and health outside of landfill use in Spain. Future efforts in stork AMR research should focus on exploring the relationship between agricultural land use and health.</p>
<p><em>What are the main take-home messages?</em></p>
<p><em>Include citations in your Rmd file using bibtex, the list of references will automatically be placed at the end</em></p>
<p>This paper <span class="citation" data-cites="leek2015">(<a href="#ref-leek2015" role="doc-biblioref">Leek &amp; Peng, 2015</a>)</span> discusses types of analyses.</p>
<p>These papers <span class="citation" data-cites="mckay2020 mckay2020a">(<a href="#ref-mckay2020" role="doc-biblioref">McKay, Ebell, Billings, et al., 2020</a>; <a href="#ref-mckay2020a" role="doc-biblioref">McKay, Ebell, Dale, et al., 2020</a>)</span> are good examples of papers published using a fully reproducible setup similar to the one shown in this template.</p>
<p>Note that this cited reference will show up at the end of the document, the reference formatting is determined by the CSL file specified in the YAML header. Many more style files for almost any journal <a href="https://www.zotero.org/styles">are available</a>. You also specify the location of your bibtex reference file in the YAML. You can call your reference file anything you like, I just used the generic word <code>references.bib</code> but giving it a more descriptive name is probably better.</p>
<div style="page-break-after: always;"></div>
</section>
</section>
<section id="references" class="level1 unnumbered">


</section>

<div id="quarto-appendix" class="default"><section class="quarto-appendix-contents" role="doc-bibliography"><h2 class="anchored quarto-appendix-heading">References</h2><div id="refs" class="references csl-bib-body hanging-indent" data-line-spacing="2" role="doc-bibliography">
<div id="ref-leek2015" class="csl-entry" role="doc-biblioentry">
Leek, J. T., &amp; Peng, R. D. (2015). Statistics. <span>What</span> is the question? <em>Science (New York, N.Y.)</em>, <em>347</em>(6228), 1314–1315. <a href="https://doi.org/10.1126/science.aaa6146">https://doi.org/10.1126/science.aaa6146</a>
</div>
<div id="ref-mckay2020" class="csl-entry" role="doc-biblioentry">
McKay, B., Ebell, M., Billings, W. Z., Dale, A. P., Shen, Y., &amp; Handel, A. (2020). Associations <span>Between Relative Viral Load</span> at <span>Diagnosis</span> and <span>Influenza A Symptoms</span> and <span>Recovery</span>. <em>Open Forum Infectious Diseases</em>, <em>7</em>(11), ofaa494. <a href="https://doi.org/10.1093/ofid/ofaa494">https://doi.org/10.1093/ofid/ofaa494</a>
</div>
<div id="ref-mckay2020a" class="csl-entry" role="doc-biblioentry">
McKay, B., Ebell, M., Dale, A. P., Shen, Y., &amp; Handel, A. (2020). Virulence-mediated infectiousness and activity trade-offs and their impact on transmission potential of influenza patients. <em>Proceedings. Biological Sciences</em>, <em>287</em>(1927), 20200496. <a href="https://doi.org/10.1098/rspb.2020.0496">https://doi.org/10.1098/rspb.2020.0496</a>
</div>
</div></section></div></main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    target: function(trigger) {
      return trigger.previousElementSibling;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->



</body></html>